File: assignments/MortgageCalculator.java
import java.util.Scanner;

public class MortgageCalculator{

public static void main (String...args){

Scanner input = new Scanner(System.in);

System.out.println("Enter the principal amount: ");

double principal = input.nextInt();

System.out.println("Enter the annual interest rate: ");

double annualRate = input.nextInt();

System.out.println("Enter the duration in years: ");

double yearlyDuration = input.nextInt();

double monthlyRate = annualRate/1200;

double timeInMonths = yearlyDuration*12;

double monthlyRateplus1 = monthlyRate +1;

double factor = (double)Math.pow(monthlyRateplus1,timeInMonths);

double monthlyPayment =principal*((monthlyRate*factor)/(factor-1));

System.out.printf("your monthly payment is %.2f",monthlyPayment);














}

}

File: assignments/mortgagecalculator.py
#mortgage calculator

principal = float(input("Enter the principal amount: "))

annual_rate= float(input("Enter the annual interest rate: "))

time_in_years= int(input("Enter the duration in years: "))

monthly_rate = annual_rate/1200

time_in_months = time_in_years*12

monthly_payment = principal*((monthly_rate*((1+(monthly_rate))**time_in_months))/(((1+monthly_rate)**time_in_months)-1))

print(round(monthly_payment,2))